TangemExchange/Models/OneInch/Targets/BaseTarget.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  BaseTarget.swift
+//  Tangem
+//
+//  Created by Pavel Grechikhin.
+//  Copyright © 2022 Tangem AG. All rights reserved.
+//
+
+import Foundation
+import Moya
+
+struct BaseTarget: TargetType {
+    let target: TargetType
+
+    var baseURL: URL {
+        target.baseURL
+    }
+
+    var path: String {
+        target.path
```

Comment:
```suggestion
        "/\(blockchain.id)/\(target.path)"
```
тогда из всех таргетов можно будет убрать эту логику. Будет проще, наверное

Reply:
✅